common/influxdb/influxdb.go

```diff
@@ -127,5 +129,13 @@ func BuildConfig(uri *url.URL) (*InfluxdbConfig, error) {
 		config.ClusterName = opts["cluster_name"][0]
 	}
 
+	if len(opts["concurrency"]) >= 1 {
+		concurrency, err := strconv.Atoi(opts["concurrency"][0])
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse `concurrency` flag - %v", err)
+		}
+		config.Concurrency = concurrency
```

Comment:
you should validate that it's greater than zero here.

Reply:
Will do.